Content.Shared/Traits/Assorted/UnrevivableComponent.cs

```diff
@@ -0,0 +1,20 @@
+namespace Content.Shared.Traits.Assorted;
+
+/// <summary>
+/// This is used for the unrevivable trait as well as generally preventing revival.
+/// </summary>
+[RegisterComponent]
+public sealed partial class UnrevivableComponent : Component
+{
+    /// <summary>
+    /// A field to define if we should display the "Genetic incompatibility" warning on health analysers
+    /// </summary>
+    [DataField]
+    public bool Analyzable { get; set; } = true;
```

Comment:
```suggestion
    public bool Analyzable = true;
```
I think these are not needed

Reply:
ok so i was right about this